style.css

```diff
@@ -21,6 +24,10 @@ nav ul {
   display: none;
 }
 
+.main-nav:hover ul {
```

Comment:
It seems like you removed your `icomoon` files.

![screen shot 2017-06-19 at 5 36 57 pm](https://user-images.githubusercontent.com/12047881/27311487-1448a5d6-5516-11e7-816d-07ad64aefa0a.png)